docs/dataset/advanced-usage/task-metrics.md

```diff
@@ -0,0 +1,76 @@
+---
+icon: kolena/metrics-glossary-16
+
+---
+
+# :kolena-metrics-glossary-20: Task Metrics
+Kolena supports automatic calculation of a number of metrics commonly used in machine learning tasks.
+
+## Regression
+Following commonly used regression metrics are supported out-of-the-box: 
+
+1. [Mean Absolute Error](../../metrics/mean-absolute-error.md)
+2. [Mean Squared Error](../../metrics/mean-squared-error.md)
+3. [Root Mean Squared Error](../../metrics/root-mean-squared-error.md)
+4. [R^2](../../metrics/coefficient-of-determination.md)
+5. Pearson Correlation
+6. Spearman Correlation
+
+Follow the steps below to setup Regression metrics: 
+<figure markdown>
+![Defining Metrics](../../assets/images/task-metrics-regression.gif)
+<figcaption>Task Metrics: Regression</figcaption>
+</figure>
```

Comment:
As discussed offline, we should include both light mode and dark mode GIFs (see images on other pages for how to flip between these two when the theme changes).

To reduce the burden of GIFs/screenshots, we can omit most of these animated GIFs. They aren't that easy to follow, and the point they make (start on QS page, open a drawer, make selections --> get metrics) is communicated by having just one GIF up top, rather than one per section.